sdk/v1/client_test.go

```diff
@@ -25,13 +25,25 @@ const (
 	serviceName   = "test-app"
 	serviceToken  = "test-token"
 	testHost      = "http://localhost:25700"
-	testJobID     = "123"
 	ifMatchHeader = "If-Match"
 	testETag      = `"56b6890f1321590998d5fd8d293b620581ff3531"`
+	testJobID     = "883c81fd-726d-4ea3-9db8-7e7c781a01cc"
+	testTaskName  = "zebedee"
 )
 
 var (
 	initialState = healthcheck.CreateCheckState(service)
+
+	expectedTask = models.Task{
+		JobID:       "883c81fd-726d-4ea3-9db8-7e7c781a01cc",
+		LastUpdated: time.Now().UTC(),
+		Links: &models.TaskLinks{
+			Self: "http://localhost:12150/jobs/abc123/tasks/zebedee",
+			Job:  "http://localhost:12150/jobs/abc123",
```

Comment:
Thanks for changing that one but please would you change the line above it too (line 41) so that there is no abc123 but instead there's the same job id value there too (883c81fd-726d-4ea3-9db8-7e7c781a01cc).

Reply:
thanks, looks good